docker-compose.yml

```diff
@@ -1,29 +1,31 @@
----
-version: '2'
 services:
+  # Run a leader node
   dkron:
-    build: .
+    image: dkron/dkron:4.0.4
```

Comment:
Use `latest` here, I don't want to keep updating this version and, more importantly, this compose should always work with latest.